liquibase-core/src/main/java/liquibase/datatype/core/TimestampType.java

```diff
@@ -121,6 +119,21 @@ public DatabaseDataType toDatabaseDataType(Database database) {
             type = new DatabaseDataType("TIMESTAMP");
         }
 
+        if (originalDefinition.startsWith("java.sql.Types.TIMESTAMP_WITH_TIMEZONE")
+            && (database instanceof PostgresDatabase
+            || database instanceof OracleDatabase
+            || database instanceof H2Database
+            || database instanceof HsqlDatabase)) {
+
+            if ((database instanceof PostgresDatabase)) {
+                type.addAdditionalInformation("WITH TIME ZONE");
+            } else {
+                type.addAdditionalInformation("WITH TIMEZONE");
```

Comment:
What `WITH TIMEZONE` is?

Reply:
I don't understand this comment. Could you add some extra details please?

Reply:
What database accepts `TIMESTAMP WITH TIMEZONE`? PostgreSQL, Oracle, H2, and HSQLDB from your PR use `TIMESTAMP [(fractionalSecondsPrecision)] WITH TIME ZONE` with the space between `TIME` and `ZONE` and this data type is a part of the SQL Standard.
https://www.postgresql.org/docs/12/datatype-datetime.html
https://docs.oracle.com/en/database/oracle/oracle-database/19/sqlrf/Data-Types.html#GUID-BE23545B-469A-4A57-8D13-505F2F5DB706
https://h2database.com/html/datatypes.html#timestamp_with_time_zone_type
http://hsqldb.org/doc/2.0/guide/sqlgeneral-chapt.html#sgc_datetime_types
And there are others, such as DB2:
https://www.ibm.com/support/knowledgecenter/en/SSEPEK_12.0.0/sqlref/src/tpc/db2z_timestampvalues.html